ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -749,10 +749,15 @@ tests:
     workflow: openshift-e2e-nutanix-ovn
   timeout: 6h0m0s
 - as: agent-integration-tests
-  commands: hack/go-integration-test.sh
+  commands: hack/go-integration-test.sh -run .AgentIntegration
```

Comment:
The integration tests suite it's meant to cover the whole product - despite the fact that it's currently used mostly for the agent-based related commands. It doesn't seem to me that an additional fragmentation will provide any additional benefit, whereas a single job that runs all the it will do (similarly to what already happens to the ut)

Reply:
I see your point, but it only runs for `agent` related changes and it's named `agent-integration-tests`. If we would like to leave it running all the `.Integration` tests, how about:

- renaming the `agent-integration-tests` job to `integration-tests`
- removing the `agent` part from https://github.com/openshift/release/pull/54406/files#diff-92fb8761a3401342e7a5678fa916ff7f552fe64f9fa412105c354e691c76a736R755 or adding the `imagebased` part there

WDYT?

Reply:
I'm also fine with leaving the `agent-integration-tests` as it is, as long as it's clear that it will also be running the `image-based-integration-tests` (which are currently being added) and also include them in any `imagebased` dir changes. Whatever you think works best for all the use cases (I'm only familiar the `image-based` and somewhat familiar with the `agent` ones) :)

Reply:
That sounds good, I'll go ahead with the suggested changes in the respective installer PR, to which I'm adding the image-based integration tests. 

In addition to these changes, I have to edit when the integration tests are run [here](https://github.com/openshift/release/pull/54406/files#diff-92fb8761a3401342e7a5678fa916ff7f552fe64f9fa412105c354e691c76a736R755).

Reply:
For future reference, the changes suggested are implemented here: https://github.com/openshift/installer/pull/8737